### ISSUE TITLE
Smaller hotkey window width when stage manager is on

### ISSUE THREAD

**Issue by brhaka** [warpdotdev/Warp#2606]
### Describe the bug

When stage manager is on, the hotkey window's width will leave a gap on the left, as other windows do by default.
The issue is that, if the hotkey window width is set to 100%, when stage manager is off it'll occupy 100% of the screen's width, but when stage manager is on it'll leave a gap to the left.

### To Reproduce

1. Enable hotkey window under Settings -> Features
2. Enable MacOS stage manager
3. Open hotkey window by pressing the configured keybinding.
4. Check the free space available on the left.

### Expected behaviour

I expect the hotkey window to occupy 100% of the screen width, with stage manager on or off, having in mind that I set the width to 100% on the settings.

### Screenshots

Stage manager **off**
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/32342284/216716853-55316e5a-21be-454d-9d05-3d1d97c683ed.png">

Stage manager **on**
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/32342284/216716941-8244b352-a690-411e-8dcd-1c0bddfa8b9a.png">

### Operating System

MacOS

### OS Version

13.1

### Shell Version

zsh 5.8.1 (x86_64-apple-darwin22.0)

### Warp Version

v0.2023.01.31.08.08.stable_02

### Does this block you from using Warp daily?

No

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
Thanks for submitting, I've noticed Hotkey window taking up the available space, so for example if the dock is visible on the screen, it takes that into account. Could be that the stage manager is taking up similar kind of space. 

We'll post any updates here.

**Comment by brhaka** [comment]
Yes, it looks like it's occupying 100% of an "inner width" instead of the screen's full width.
I'll pay attention to any updates posted here!
Thanks.

**Comment by Geddard** [comment]
Can confirm, it's pretty annoying xD

**Comment by dohooo** [comment]
Hi, are there any updates?

**Comment by Geddard** [comment]
Almost a year later, this still looks bad :/

**Comment by jakobfdev** [comment]
+1
a fix would be great.
Tools like "Rectangle" tackled the same problem. If there's been no research into the topic, maybe their free version's open source code might help
https://github.com/rxhanson/Rectangle/issues?q=is%3Aissue+stage+manager

**Comment by FrenchBen** [comment]
Seems to be native to stage manager and how they expose "100%" to the apps. A similar bug exists, if you try to minimize Warp to stage manager, the Warp window will be stuck in the middle of the screen, in its minimized form

**Comment by AlexisZankowitch** [comment]
I'm using state manager as well and I didn't have the issue ... until I played with the dock settings. It seems the size issue only occurs for me when the Dock setting: **show windows from an application** is set to `all at once`. The issue doesn't occur when this setting is set to `one at a time`

- state manager is activated 
- Dock setting: show windows from an application set to `all at once` 
- Use hotkey to open Warp
- expand left side of the window 
- Hide Warp
- Use hotkey to open Warp

- strange behaviour
https://github.com/warpdotdev/Warp/assets/12337798/ed14534a-b7d0-48cf-b42d-bdeb3ef09b3d
